style/ember/README.md

```diff
@@ -2,3 +2,8 @@ Ember
 =====
 
 * Don't put a space between the opening handlebars braces and the variable.
+
+Ember-Data
+----------
+
+* Visually separate relationships (`DS.{hasMany,belongsTo}`) from attributes (`DS.attr`).
```

Comment:
Since there's no example provided inline, can this be more explicit? Like `Separate relationships with newline`?